### ISSUE TITLE
Delete old transcription entries in Andean database

### ISSUE THREAD

**Issue by PaulHeggarty** [lingdb/Sound-Comparisons#315]
Remove old doubled-up transcriptions from languages, especially for the Andes (but also a few Romance cases).

I'm pretty sure that this is because my original entries have AlternativePhoneticRealisationIx=1, while all the new ones have AlternativePhoneticRealisationIx=0.
 
So, please delete ALL records in the transcriptions table for the Andes that have this combination of three values:

StudyIx=2     (i.e. Andean)

FamilyIx=1  OR  2   OR  3     (don't delete those with FamilyIx=8, i.e. Mapudungun, because we don't have a problem here, and I think it's all fine)

AlternativePhoneticRealisationIx=1

More information on the Romace cases to delete will follow.

**Comment by runjak** [comment]
I have deleted the according Transcriptions.
I'll make sure to fetch a current copy of the database from the server to have a good copy on my machine and afterwards close this issue.
The query I've used to delete the records in question looks like this:
```
DELETE FROM Transcriptions_Andean
WHERE StudyIx = 2
  AND FamilyIx IN (1,2,3)
  AND AlternativePhoneticRealisationIx = 1;
```
There is an official documentation of the grammar that the queries must follow for MySQL: [dev.mysql.com](https://dev.mysql.com/doc/refman/5.7/en/delete.html)
If you'd like a resource to get into using SQL more extensively I can recommend [sql-island.de](http://wwwlgis.informatik.uni-kl.de/cms/courses/informationssysteme/sqlisland/).
